integration/integration_test.go

```diff
@@ -30,6 +30,8 @@ var (
 	image = flag.String("image", "", "gadget container image")
 
 	doNotDeploy = flag.Bool("no-deploy", false, "don't deploy Inspektor Gadget")
```

Comment:
nit: Maybe changing `doNotDeploy` to `doNotDeployIG` would make the code clearer now that we also have `deploySPOFlag`?